src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        Random randomColors = new Random();
```

Comment:
Instead of creating random field as local var in method each of it's invocation, make it object final variable